packages/api/src/external/carequality/document/shared.ts

```diff
@@ -51,8 +54,8 @@ export const cqToFHIR = (
   };
 
   const containedResources = docRef.authorInstitution
-    ? generateCQFHIRContained(docRef.authorInstitution)
-    : undefined;
+    ? mapToContainedOrganization(docRef.authorInstitution)
+    : mapToContainedOrganization(orgName);
```

Comment:
I might be wrong, but I think we can have both author and organization.

AFAIK, we should build contained in a way it can contain multiple resources. Likely need to avoid dups, though.

Lastly, DocumentReference has a specific field called `author` - should we not use it for `authorInstitution`?

Reply:
These seem like nice-to-haves, while this PR is addressing an issue in prod affecting a CX

Reply:
I don't think adding the correct information to the Doc Ref is a nice-to-have. We might be losing information with the ternary (single contained approach).